src/data/Title.js

```diff
@@ -0,0 +1,21 @@
+import chipCookie from "@/images/chip.webp";
+import heartCookie from "@/images/heart.webp";
+import swirlCookie from "@/images/swirl.webp";
+
+export const COOKIES = {
+  chip: chipCookie,
+  heart: heartCookie,
+  swirl: swirlCookie,
+};
+
+export const COLORS = {
+  red: {
+    light: "text-red-100",
```

Comment:
check out tailwind config file for the colors

"text-biscuits-red-100" for example.